certified-connectors/Cyberday/apiDefinition.swagger.json

```diff
@@ -332,6 +332,92 @@
         ],
         "description": "Create a System in your Cyberday account."
       }
+    },
+    "/api/external/systems/topics/advanced/": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@tomagendium empty schema is not allowed. Please remove or define a proper schema.
Please fix all empty schemas in your connector swagger file. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).